bpython/curtsies.py

```diff
@@ -122,6 +122,9 @@ def after_suspend():
                 window.__enter__()
                 interrupting_refresh()
 
+            def get_top_usable_line():
+                return window.top_usable_row
+
```

Comment:
Is this is the sort of thing you're thinking off? https://github.com/bpython/bpython/compare/inheritance-over-composition